admin.html

```diff
@@ -200,8 +200,8 @@ <h1 class = "class_name" style = "text-align: center; margin-top:-5px;"></h1>
         <div class="col-md-12 col-sm-12">
           <h3> Settings </h3>
           <h4> Change Password </h4>
-          <p class="error error_password_mismatch" hidden> Passwords do not match <br/> </p> 
-          <p class="error error_password_incorrect" hidden> Incorrect password <br/> </p> 
+          <p class="error error_password_mismatch" hidden> Passwords do not match <br/> </p>
```

Comment:
If text content of a node is in a single line, there shouldn't be spaces around it.